index.html

```diff
@@ -0,0 +1,211 @@
+                <div class="theday">
+                    <span class="day-name">Sun</span>
+                    <span class="day-date">23</span>
+                </div>
+                <div class="time">
+                    <span class="date-start">18:00</span>
+                    <span class="date-end">22:00</span>
+                    <span class="d-block">UTC+1</span>
+                </div>
+                <div class="location"> Tirana Reja</div>
+                <div class="running-order">
+                    <h5>Bojken Lako</h5>
+                </div>
+                <div class="ticket-wrapper">
+                    <a href="#" class="t-btn">Ticket</a>
+                </div>
+            </div>
+            <div class="row-event">
+                <div class="theday">
+                    <span class="day-name">Mon</span>
+                    <span class="day-date">24</span>
+                </div>
+                <div class="time">
+                    <span class="date-start">20:00</span>
+                    <span class="date-end">22:00</span>
+                    <span class="d-block">UTC+1</span>
+                </div>
+                <div class="location"> Tirana Bllok</div>
+                <div class="running-order">
+                    <h5>Genc Salihu</h5>
+                </div>
+                <div class="ticket-wrapper">
+                    <a href="#" class="t-btn">Ticket</a>
+                </div>
+            </div>
+        </div>
+        <a class=" hidden bands" href="#">See the Bands in action </a>
+        <div class="btn-container">
+            <button class="main-btn" type="button">
+                Join our telegram channel
+            </button>
+        </div>
+
+    </section>
+    <!--Features Section-->
+    <section class="features">
+        <div class="features-heading">
+            <h2>Features </h2>
+            <h3>Special Projects and Collaborations</h3>
+            <div class="features-line"></div>
+        </div>
+        <div class="feature-container"></div>
+        <div class="more-btn">
+            <button class="more" type="button">MORE
+                <img id="down" src="images/down-arrow.png" alt="">
+            </button>
+        </div>
+
+        <div class="less-btn">
+            <button class="less" type="button"> LESS
+                <img id="up" src="images/arrow-up.png" alt="">
+            </button>
+        </div>
+    </section>
+
+    <section class="partner" id="sponsor">
+        <h2 class="partnertitle">Partners</h2>
+        <div class="main-heading">
+            <div class="main-line"></div>
+        </div>
+        <div class="partner-logos">
+            <img id="logo1" src="images/Tob-logo.svg" alt="">
+            <img id="logo2" src="images/Logo-Bashkia-Tirane.svg" alt="">
+            <img id="logo3" src="images/birra-korca-logo2-01.svg" alt="">
+            <img id="logo4" src="images/Posta_Shqiptare.svg" alt="">
+        </div>
```

Comment:
- [ ] [OPTIONAL] For this section you could use a `ul` tag and your inner divs become `li` of the `ul`  it could be something like this :point_down: 
    ```HTML 
                                    <ul>
                                         <img id="logo1" src="images/Tob-logo.svg" alt="">
                                         <img id="logo2" src="images/Logo-Bashkia-Tirane.svg" alt="">
                                   </ul>
     ```